lang/en/tool_mergeusers.php

```diff
@@ -194,3 +194,9 @@
 $string['finishtime'] = 'Finished merging at {$a}';
 $string['timetaken'] = 'Merge took {$a} seconds';
 $string['privacy:metadata'] = 'The Merge User Accounts plugin does not store any personal data.';
+$string['queue_to_process_pending_merge_requests'] = 'Get queue of the pending merge requests and launch adhoc task for each single request';
+$string['cannotfinduser'] = 'Cannot find the user with {$a->userfield} {$a->uservalue}. Change the input parameters.';
```

Comment:
There will not be possibility for the adhoc task to `change the input parameters`. I think we should remove this sentence from the string `cannotfinduser`.

Reply:
ok